### DISCUSSION TITLE
How Can I change the settings for the protocol and port?

### DISCUSSION THREAD

**Discussion by uo-gimrt** [mozilla/hubs#3091] [Hubs Cloud Questions] [answered]
I would like to use port 443 for URL https communication.
I checked the company's proxy logs and found that the https communication used by Hubs Cloud included communication on a port other than port 443 (specifically port 80).
My company's proxy server is not using any https communications other than port 443 for communications on https.
I'm set to reject it, so I get a connection error.

If there's a place in hubs cloud where I can configure the protocol and port settings.
Protocol→https
Port → 443
Please tell me how to change the settings as follows.

**Comment by robinkwilson** [comment, marked as answer]
Protocol is both https + wss even on port 80.

Here's the issue for customizing ports: https://github.com/mozilla/hubs-cloud/issues/90
Here is the related discussion thread: https://github.com/mozilla/hubs/discussions/3043

We currently don't have a way to customize the port configuration, but we know that's important for strict firewalls and are working on it!!
